packages/components/Organisms/Grants/GrantsOverview.vue

```diff
@@ -96,8 +95,10 @@ export default {
               return terms.ongoing
             }
           }
+          var isBetween = require('dayjs/plugin/isBetween')
+          dayjs.extend(isBetween)
           // displays status as "Opening on startdate" when current date is within one month of startdate
-          if (now.isBetween(moment(start).subtract(1, 'months'), start)) {
```

Comment:
@yeniatencio have you checked that this is functioning as expected ? Not sure if this has proper tests (my bad it should) but I'm pretty sure this won't work without also importing the `isBetween` plugin from dayJs https://day.js.org/docs/en/plugin/is-between

Reply:
OK @dylankelly, I have added the import instead of var 'isBetween = require('dayjs/plugin/isBetween')'. Both working as expected.